server/http/src/server.rs

```diff
@@ -49,7 +50,8 @@ pub struct GraphQLServer {
     query_sink: Option<Sender<Query>>,
     schema_event_sink: Sender<SchemaEvent>,
     store_event_sink: Sender<StoreEvent>,
-    schema: Arc<Mutex<Option<Schema>>>,
+    // Maps a graph name to it's schema.
```

Comment:
I think you mean `subgraph name` here. Can we make this `Maps subgraph names and IDs to their schemas`? I'll describe why further down.